src/JMS/Serializer/Annotation/AccessorFinder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Copyright 2016 Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace JMS\Serializer\Annotation;
+use JMS\Serializer\Accessor\Finder\DefaultAccessorFinder;
+
+/**
+ * Defines accessor finder.
+ *
+ * @Annotation
+ * @Target("CLASS")
+ */
+
+final class AccessorFinder
+{
+    public $class = DefaultAccessorFinder::class;
```

Comment:
What about having also a "service" ? (and some kind of accessor registry where is possible to add accessors via dependency injection?) to avoid instantiating the class in the metadata reader.... in this way 
accessor finders can have parameters and their dependencies too.